### PR TITLE
[PLATIR-29075] https, propositionFetch, updated docs, latest alloy build, and more

### PR DESCRIPTION
**Pull request by jasonwaters**
## Description

This PR is in response to the following customer issue: https://jira.corp.adobe.com/browse/PLATIR-29075

- Update Strategy for Analytics to clearly call out that target must be enabled for both datastreams.
- Now using `decisioning.propositionFetch` as eventType for server-side personalization requests. 
- Switch to `uuidv4` for request ids sent.  (uuidv1 ids were not showing up in splunk logs)
- upgrade alloy to `v2.16.0` in all samples
- all samples now serve content over `https`.  This change was made due to consultants/customers having difficulty using VEC with the sample.  Using https prevents insecure content errors.
- ☝️ requires `mkcert` to be installed.  I added documentation for it and updated each readme with a step to Setup local SSL certificates for https
- Formatted all source files using prettier.  So you'll see subtle formatting changes in many files.

### CODE REVIEW

File: AnalyticsStrategy.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+The strategy employed by these samples utilizes two different datastreams. One with analytics configured, and one without. The datastream _**without**_ analytics configured is used for edge requests sent server-side. And the datastream _**with**_ analytics configured is used for alloy on the client-side. That way the server-side request does not register any analytics events, but the client side does. The result is analytics are accurately counted only once.
 
 ### Flow Diagram
 
-| Datastream   | Analytics configured |
-|--------------|----------------------|
-| Datastream 1 | No                   |
-| Datastream 2 | Yes                  |
-
+| Datastream   | Analytics configured | Target configured |
```
Comment (jfkhoury) [inline AnalyticsStrategy.md:9]: Nice!

File: LocalSSLCertificateSetup.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Local SSL Certificates
```
Comment (jfkhoury) [inline LocalSSLCertificateSetup.md:1]: Nice as well 👍

**Review by jfkhoury — APPROVED** [review]
Looks great, thank you for all the changes. Should we run it by Bennet or Daniel to test the ssl stuff and end 2 end workflow?

**Review by jonsnyder — APPROVED** [review]
This is great. I need to create a top and bottom of page sample.

**Pull request merged**